File: 0268-missing-number/0268-missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int n = nums.size();
        int t_sum = n * (n + 1) / 2;
        int ele_sum = 0;

        for (int num : nums) {
            ele_sum += num;
        }

        return t_sum - ele_sum;
    }
};
    

File: 0268-missing-number/0268-missing-number.py
class Solution(object):
    def missingNumber(self, nums):
        n = len(nums)
        t_sum = n * (n + 1) // 2  # Sum of n numbers 
        ele_sum = sum(nums)       # Sum of elements in array
        return t_sum - ele_sum
        
        